Update registration steps for Entra ID app registration (#1499)
MicrosoftDocs/azure-dev-docs: articles/java/identity/enable-spring-boot-webapp-authentication-entra-id.md

1. Navigate to the [Azure portal](https://portal.azure.com) and select **Microsoft Entra ID**.

1. Select **App registrations** on the navigation pane, then select **New registration**.

1. In the **Register an application page** that appears, enter the following application registration information:

   - In the **Name** section, enter a meaningful application name for display to users of the app - for example, `java-spring-webapp-auth`.

>    - Under **Supported account types**, select **Single tenant only - TENANT_NAME**.

Author: ```suggestion
   - Under **Supported account types**, select **Single tenant only - TENANT_NAME** (`TENANT_NAME` varies per tenant).
```

1. Type a description - for example, **app secret**.

> 1. Select one of the available durations: **Recommended: 180 days (6 months)**, **90 days (3 months)**, **365 days (12 months)**, **545 days (18 months)** or **730 days (24 months)**.

Author: The list reflects exactly the possible dropdown options.

1. Find the placeholder `Enter_Your_Tenant_ID_Here` and replace the existing value with your Microsoft Entra tenant ID.

> 1. Find the placeholder `Enter_Your_Client_ID_Here` and replace the existing value with the application ID or `clientId` of the `java-spring-webapp-auth` app registration copied from the Azure portal.

Author: It's as intended - disagree with the review comment

> 1. Find the placeholder `Enter_Your_Client_ID_Here` and replace the existing value with the application ID or `clientId` of the `java-spring-webapp-auth` app registration copied from the Azure portal.

Reviewer: It seems like you're trying to solve a problem similar to the one that Copilot is pointing out, i.e. that it isn't the "app" OR the "registration" that was copied, but just the ID itself, right? How about something like this:
 
```suggestion
1. Find the placeholder `Enter_Your_Client_ID_Here` and replace the existing value with the `java-spring-webapp-auth` app's application ID or `clientId` that you copied from the Azure portal.
```
